profiles/logical/hpos/default.nix

```
@@ -151,6 +151,11 @@ in
           proxyPass = "http://127.0.0.1:4656";
           proxyWebsockets = true;   # TODO: add proxy_send_timeout, proxy_read_timeout HERE
         };
+
+         "/trycp/" = {
+          proxyPass = "http://127.0.0.1:9000";
+          proxyWebsockets = true; 
```

```suggestion
          proxyWebsockets = true; 
          extraConfig = ''
            proxy_send_timeout 1d;
            proxy_read_timeout 1d;
          '';
```
trycp is gonna be working on connections open for a long time. We need these two directives so that connection does not drop.